lightning/src/ln/channelmanager.rs

```diff
@@ -4353,8 +4355,6 @@ where
 				// we don't have the channel here.
 				return Err(("Refusing to forward over real channel SCID as our counterparty requested.", 0x4000 | 10));
```

Comment:
Do we perform this check for trampoline forwards, once we know the outgoing channel?